lib/node_modules/@stdlib/iter/cartesian-product/test/test.js

```diff
@@ -0,0 +1,225 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2019 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require('tape');
+var toAccessorArray = require('@stdlib/array/base/to-accessor-array');
+var iterCartesianProduct = require('./../lib');
+
+
+// TESTS //
+
+tape('main export is a function', function test(t) {
+	t.ok(true, __filename);
+	t.strictEqual(typeof iterCartesianProduct, 'function', 'main export is a function');
+	t.end();
+});
+
+tape('the function throws an error if provided a first argument which is not a collection', function test(t) {
+	var values;
+	var i;
+
+	values = ['5', 5, NaN, true, false, null, void 0, {}, function noop() {}];
```

Comment:
```suggestion
	values = [
	    '5', 
	    5, 
	    NaN, 
	    true, 
	    false, 
	    null, 
	    void 0, 
	    {}, 
	    function noop() {}
	];
```
Please format lines that are long

Reply:
This is still not resolved